litmus-portal/frontend/nginx/nginx.conf

```diff
@@ -25,4 +25,20 @@ server {
     #location ~ /\.ht {
     #    deny  all;
     #}
+
+    location /auth/ {
+        proxy_set_header   Host                 $host;
+        proxy_set_header   X-Real-IP            $remote_addr;
+        proxy_set_header   X-Forwarded-For      $proxy_add_x_forwarded_for;
+        proxy_set_header   X-Forwarded-Proto    $scheme;
+        proxy_pass "http://litmusportal-server-service:9003/";
+    }
+    
+    location /api/ {
```

Comment:
Path should be /query Right?

Reply:
Actually the path to grapql is `/api/query`  `/api/` gives the gui for constructing queries and mutations.